.gitignore

```diff
@@ -74,7 +74,7 @@ typings/
 
 # dotenv environment variables file
 .env
```

Comment:
I think .env should be commited to git, to be used as the default
this way the code will run even if a developer does not have any env values

Reply:
yeah, agreed, I'll add that shortly.

Reply:
Fixed.